packages/tasks/src/local-apps.ts

```diff
@@ -42,8 +42,29 @@ export type LocalApp = {
 	  }
 );
 
-function isGgufModel(model: ModelData) {
-	return model.tags.includes("gguf");
+function isGgufModel(model: ModelData): boolean {
+    return model.config?.quantization_config?.quant_method === "gguf";
```

Comment:
in case some repo doesn't have a quantization config file
```suggestion
    return model.config?.quantization_config?.quant_method === "gguf" || model.tags.includes("gguf");
```

Reply:
does vLLM support all GGUF files @simon-mo?

Reply:
@julien-c it's not used by the [vLLM integration](https://github.com/huggingface/huggingface.js/pull/693/files#diff-98bf76e04efd0b5b0e0ea40e3d8d8ade7b129f461e2f7ff45ff3ff6e567c9302R142) but it's used by other apps and was modified in the meantime (see https://github.com/huggingface/huggingface.js/pull/693/commits/43de0e9d71804d330f6a4c17c3ac0366df93435d)

Reply:
ah ok!